perllib/FixMyStreet/Script/ArchiveOldEnquiries.pm

```diff
@@ -101,7 +103,12 @@ sub get_closure_message {
         my $file = path($opts->{closure_file});
         chomp(my $message = $file->slurp_utf8);
         return $message;
-
+    } else {
+        my $d = dirname(File::Spec->rel2abs($0));
+        my $file = path("$d/../templates/web/default/archive-old-enquiries.txt");
+        chomp(my $message = $file->slurp_utf8);
+        $message =~ s/\[\[Council_name\]\]/$opts->{council_name}/g;
```

Comment:
Seems odd not to use all the standard template handling code for this template! You'd create a `FixMyStreet::Template` object and call process on it, see few other examples in Peterborough or Email.pm